src/WebJobs.Script.WebHost/Global.asax.cs

```diff
@@ -78,19 +78,28 @@ protected void Application_Error(object sender, EventArgs e)
             EventGenerator eventGenerator = new EventGenerator();
 
             // Get the exception object.
-            Exception exc = Server.GetLastError();
+            Exception unhandledEx = Server.GetLastError();
             string subscriptionId = Utility.GetSubscriptionId() ?? string.Empty;
             string appName = Utility.GetWebsiteUniqueSlotName() ?? string.Empty;
+            string exStackTrace = string.Empty;
+            string exType = string.Empty;
+            string exMessage =  string.Empty;
+            if (unhandledEx != null)
+            {
+                exStackTrace = Sanitizer.Sanitize(unhandledEx.ToFormattedString());
+                exType = unhandledEx.GetType().ToString();
+                exMessage = unhandledEx.Message;
```

Comment:
Can any of these values be null. Do we need to check for null values within.

Reply:
unlikely. But added more checks just to be safe. Thanks!